provisioning_templates/snippet/kickstart_ifcfg_generic_interface.erb

```diff
@@ -42,6 +42,10 @@ DEFROUTE=<%= primary %>
 <%- end -%>
 <%- if @interface.virtual? && ((!@subnet.nil? && @subnet.has_vlanid?) || @interface.vlanid.present?) -%>
 <%=   "VLAN=yes" %>
+<%-   if @interface.identifier.match(/^vlan/) -%>
```

Comment:
would it make sense to check for `/vlan\d+/`? just in case someone used vlan5.10 (not sure what would that mean :-)